stacks-node/src/tests/epoch_21.rs

```diff
@@ -2894,7 +2885,6 @@ fn test_pox_reorg_one_flap() {
 
 /// PoX reorg tests where two miners take turn mining hidden anchor blocks.
 /// Both miners mine in the reward phase, and in doing so, confirm their hidden anchor blocks.
-/// The heaviest affirmation map grows as n+pppa+
```

Comment:
I'm not even sure this test _could_ pass anymore.  This test exercises the affirmation map system.